### ISSUE TITLE
Don't fill in generated enum fields with _

### ISSUE THREAD

**Issue by DomWilliams0** [intellij-rust/intellij-rust#6035]
## Environment

* **IntelliJ Rust plugin version:** 0.3.129.3308-202
* **Rust toolchain version:** 1.46.0
* **IDE name and version:** CLion 2020.2.1
* **Operating system:** Linux

It's a minor inconvenience when the `Add remaining patterns` intention fills in all struct fields as unused, like this:

```rust
enum Enum {
    A,
    B {
        c: u32,
        d: String,
    }
}

match ... {
    Enum::A => {},
}

// use "Add remaining patterns"
match ... {
    Enum::A => {},
    Enum::B { c: _, d: _ } => {}
}
```
I think it would be better for the `Enum::B` variant to be generated with either all of its fields (`{c, d}`) or none (`{ .. }`), otherwise you *always* have to go back and modify the generated code.

Interestingly the `Fill match arms` intention already generates `{ .. }`, so maybe this is unintentional?

**Comment by Undin** [comment]
Fixed in #5720
@DomWilliams0 note, `Fill match arms` intention was superseded by `Add remaining patterns` quick fix which supports match expressions without any arm now

**Comment by DomWilliams0** [comment]
@Undin Fantastic, thank you
